### PR TITLE
Upgrade patches from Chromium 78.0.3904.50 to Chromium 78.0.3904.70 (uplift to 0.71.x)

### PR DESCRIPTION
**Pull request by bsclifton**
Uplift of https://github.com/brave/brave-browser/pull/6571
Fixes https://github.com/brave/brave-browser/issues/6508
Related https://github.com/brave/brave-browser/pull/6576

Approved, please ensure that before merging: 
- [ ] You have checked CI and the builds, lint, and tests all pass or are not related to your PR. 
- [x] You have tested your change on Nightly. 
- [x] The PR milestones match the branch they are landing to. 

After you merge: 
- [x] The associated issue milestone is set to the smallest version that the changes is landed on.

### CODE REVIEW

**Comment by bsclifton** [comment]
CI looks good, except for failure on build in Android. Restarting that one...

File: patches/chrome-browser-android-chrome_feature_list.cc.patch [hunk 1]
```diff
@@ -0,0 +1,12 @@
+diff --git a/chrome/browser/android/chrome_feature_list.cc b/chrome/browser/android/chrome_feature_list.cc
+index bfeea2227a320b1580868a64875d7497b8455fdb..cee091d8c6f409febd66a3f4ec42fa3cebf2b24e 100644
+--- a/chrome/browser/android/chrome_feature_list.cc
++++ b/chrome/browser/android/chrome_feature_list.cc
+@@ -231,6 +231,7 @@ const base::Feature* kFeaturesExposedToJava[] = {
+ };
+ 
+ const base::Feature* FindFeatureExposedToJava(const std::string& feature_name) {
++  BRAVE_FIND_FEATURE_EXPOSED_TO_JAVA
```
Comment (yrliou) [inline patches/chrome-browser-android-chrome_feature_list.cc.patch]: https://github.com/brave/brave-core/commit/8ecdac337997f94a20227a541bc24515849a0cc3
This patch is added in 0.72.x and shouldn't be uplifted.

Comment (yrliou) [reply patches/chrome-browser-android-chrome_feature_list.cc.patch]: That's why the android build failed as follows.
```
01:20:19  ../../chrome/browser/android/chrome_feature_list.cc:234:3: error: use of undeclared identifier 'BRAVE_FIND_FEATURE_EXPOSED_TO_JAVA'
01:20:19    BRAVE_FIND_FEATURE_EXPOSED_TO_JAVA
01:20:19    ^
01:20:19  1 error generated.
```

Comment (bsclifton) [reply patches/chrome-browser-android-chrome_feature_list.cc.patch]: oh no! Can you help me remove it? That would explain the Android failure 😅

Comment (yrliou) [reply patches/chrome-browser-android-chrome_feature_list.cc.patch]: sure, on it.

Comment (yrliou) [reply patches/chrome-browser-android-chrome_feature_list.cc.patch]: https://github.com/brave/brave-core/pull/3774/commits/e7a1361a81247e6b08d3b78a2fee88999fa6737f

**Comment by bsclifton** [comment]
Lint failed, but everything else looks good! Thanks for the assist, @yrliou 😄👍

**Comment by bsclifton** [comment]
I used admin merge for this, but had gotten prior approval from @kjozwiak, @srirambv, and @LaurenWags (in that order!)

**Pull request merged**